spec/datashake-ruby-sdk/review_index/v1/profiles_spec.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Datashake::ReviewIndex::V1::Profiles do
+  let(:client) do
+    Datashake::ReviewIndex::V1.new(
+      Datashake::ReviewIndex::Client.new(token: "abcdef")
+    )
+  end
+
+  subject { described_class.new(client) }
+
+  context "when request timeouts" do
+    it "throwns an error" do
+      connection_mock = double
+      allow(connection_mock).to receive(:get).and_raise(Faraday::TimeoutError)
+
+      client = instance_double(
+        Datashake::ReviewIndex::Client,
+        token: "abcdef",
+        connection: connection_mock
+      )
+
+      version = Datashake::ReviewIndex::V1.new(client)
+      subject = described_class.new(version)
+
+      expect do
+        subject.callback("https://xmpl.com/cb").name("KFC").fetch
+      end
+        .to raise_error { |error| expect(error).to be_a(Faraday::TimeoutError) }
+    end
+  end
+
+  context "when token is invalid" do
+    it "throwns an error", :aggregate_failures do
+      VCR.use_cassette("review_index/v1/profiles/invalid_token") do
```

Comment:
Btw, should I move ReviewScraper cassettes to "review_scraper/v2" instead of just "v2"?

Reply:
Done